config/routes.rb

```diff
@@ -1,4 +1,11 @@
+require "sidekiq/web"
```

Comment:
One tiny stylistic thing – this `require` should have a blank line after it

Reply:
Have added in the stylistic blank line